internal/api/utils.go

```diff
@@ -33,3 +37,9 @@ func StringPtrSliceToStringSlice(s []*string) []string {
 	}
 	return out
 }
+
+// Check that a string conforms to the UUID format
+func IsValid(u string) bool {
+	_, err := gofrsuuid.FromString(u)
```

Comment:
We're already importing another UUID module "`github.com/google/uuid`" so we should stick to one -- whichever is more capable.  I don't know much about "`github.com/gofrs/uuid`", but Google's module has a [Validate](https://pkg.go.dev/github.com/google/uuid#Validate) function that seems like it would work here.